### PR TITLE
Getter for full verification key

### PR DESCRIPTION
**Pull request by StanislavBreadless**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/contracts/state-transition/chain-interfaces/IVerifier.sol [hunk 1]
```diff
@@ -9,6 +9,30 @@ struct VerifierParams {
...
+}
+
+/// @notice A struct that represents the verification key data.
+struct VerificationKeyData {
+    /// @notice 8 points for gate setups
+    ECPoint[8] gateSetup;
+    /// @notice 2 points for gate selectors
+    ECPoint[2] gateSelectors;
+    /// @notice 4 points for permutations
+    ECPoint[4] permutation;
```
[the remark this answers is not in the document]
Comment (StanislavBreadless) [reply l1-contracts/contracts/state-transition/chain-interfaces/IVerifier.sol:25]: not sure, `setup` I believe should be in singular. Permutations not sure cc @olesHolem

[the remark this answers is not in the document]
Comment (olesHolem) [reply l1-contracts/contracts/state-transition/chain-interfaces/IVerifier.sol:25]: These are four distinct polynomials that define one permutation. I think `permutationPolys` and `gateSetupPolys` could be a good naming

[the remark this answers is not in the document]
Comment (olesHolem) [reply l1-contracts/contracts/state-transition/chain-interfaces/IVerifier.sol:25]: PS in the Plonk whitepaper they are also mentioned like `permutation
polynomials`

**Comment by StanislavBreadless** [comment]
The feature that required this change has been removed from the timeline, so I close this PR as well

**Pull request closed without merging**